`make check` fails under mono 4.0

Mono 4.0 removes the 3.5 build targets that Stylecop (or maybe stylecopplus?) is compiled against. I expect this will break travis once the official release is pushed to the repos.

```
System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Framework, Version=3.5.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies.
File name: 'Microsoft.Build.Framework, Version=3.5.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
  at OpenRA.Mods.Common.UtilityCommands.CheckCodeStyle.Run (OpenRA.ModData modData, System.String[] args) [0x00087] in /Users/paul/src/OpenRA/OpenRA.Mods.Common//UtilityCommands/CheckCodeStyle.cs:39 
  at OpenRA.Utility.Program.Main (System.String[] args) [0x0015a] in /Users/paul/src/OpenRA/OpenRA.Utility//Program.cs:54 
[ERROR] FATAL UNHANDLED EXCEPTION: System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Build.Framework, Version=3.5.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies.
File name: 'Microsoft.Build.Framework, Version=3.5.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
  at OpenRA.Mods.Common.UtilityCommands.CheckCodeStyle.Run (OpenRA.ModData modData, System.String[] args) [0x00087] in /Users/paul/src/OpenRA/OpenRA.Mods.Common//UtilityCommands/CheckCodeStyle.cs:39 
  at OpenRA.Utility.Program.Main (System.String[] args) [0x0015a] in /Users/paul/src/OpenRA/OpenRA.Utility//Program.cs:54 
make: *** [check] Error 1
```

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/12001271-make-check-fails-under-mono-4-0?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

--- a/OpenRA.Utility.exe.config
+++ b/OpenRA.Utility.exe.config
@@ -1,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1" appliesTo="v1.0.3705">
+      <dependentAssembly>
+        <assemblyIdentity name="Microsoft.Build.Framework" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-100.0.0.0" newVersion="12.0.0.0" />
+      </dependentAssembly>
+    </assemblyBinding>
+  </runtime>
+</configuration>

